chainercv/transforms/image/pca_lighting.py

```diff
@@ -0,0 +1,47 @@
+import numpy
+
+
+def pca_lighting(img, sigma, eigen_value=None, eigen_vector=None):
+    """Alter the intensities of input image using PCA.
+
```

Comment:
Make "AlexNet style color augmentation" as the title of docstring, and add a line below that describes what it is (e.g. adding random amount of perturbation in the direction of principle components).